src/styles/bank_transactions.scss

```diff
@@ -736,11 +730,21 @@
 }
 
 @container (min-width: 1400px) {
-  .Layer__bank-transactions__date-col {
-    min-width: 160px;
-    width: 160px;
-    max-width: 160px;
+  .Layer__table.Layer__bank-transactions__table {
+    .Layer__table-header.Layer__bank-transactions__date-col {
+      min-width: 160px;
+      width: 160px;
+      max-width: 160px;
+      padding-left: var(--spacing-xl);
+      padding-right: var(--spacing-xl);
+    }
+
+    .Layer__table-header.Layer__table-cell__category-col {
+      padding-left: var(--spacing-xl);
+      padding-right: var(--spacing-xl);
+    }
```

Comment:
Here is the only relevant change. In overall, we don't want extra paddings on large screens, except Bank Transactions table